src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1137,6 +1157,18 @@ private ExtBidResponse toExtBidResponse(List<BidderResponse> results, BidRequest
                                 bidderCatalog.errorForDeprecatedName(bidder)))));
     }
 
+    private Map<String, List<ExtBidderError>> putPrebidErrors(List<String> errors) {
```

Comment:
Please rename method to something along the lines of `extractCacheErrors`, argument is better to be renamed to `cacheErrors`